talkmeup/templates/home.html

```diff
@@ -2,83 +2,195 @@
 
 {% block content %}
 
-  <div class="embed-responsive embed-responsive-16by9">
-    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/Tiy2LONr050?autoplay=1&loop=1&playlist=Tiy2LONr050&mute=1"></iframe>
-  </div>
+<div style="text-align: center">
+<!--  <div class="embed-responsive embed-responsive-16by9">-->
+    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/Tiy2LONr050?autoplay=1&loop=1&playlist=Tiy2LONr050&mute=1" height="550" width="970" ></iframe>
```

Comment:
height="550" width="970"
this hardcodes height and width, will it still be "responsive"?